### PR TITLE
feat(1555): Pass in UI URI to launcher

### PR DESCRIPTION
**Pull request by tkyi**
Related to https://github.com/screwdriver-cd/launcher/pull/265
Issue: https://github.com/screwdriver-cd/screwdriver/issues/1555

### CODE REVIEW

**Review by minzcmu — APPROVED** [review]

**Comment by minzcmu** [comment]
hmm it seems like this has to go out together with executor-k8s-vm. We should ping yahoo japan once we merge them

File: scripts/hyper-runner.sh [hunk 1]
```diff
@@ -105,6 +107,10 @@ if [[ -z "$STORE_URI" ]]; then
   log "--store_uri is a required argument";
   printUsage;
 fi
+if [[ -z "$UI_URI" ]]; then
```
Comment (minzcmu) [inline scripts/hyper-runner.sh:110]: shall we relax the check here? so that UI_URI is optional?

Comment (tkyi) [reply scripts/hyper-runner.sh:110]: should we assign a default or just ignore if missing?

Comment (tkyi) [reply scripts/hyper-runner.sh:110]: Added default 👍

**Review by minzcmu — APPROVED** [review]

**Pull request merged**